components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/JsGraphBuilder.java

```diff
@@ -501,6 +504,24 @@ public String loadLocalLibrary(String functionLibraryName) throws FunctionLibrar
         return libraryScript;
     }
 
+    /**
+     * Get the secret by name.
+     *
+     * @param secretName secretName
+     * @return secretValue
+     * @throws SecretManagementException
```

Comment:
complete the comment

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/5723/commits/a65ec8c708a4d8a3692b43d9a953b697487c1fce